### PR TITLE
feat: update readme info, add extraEnvironmentVariables to datadog and sentry

### PR DESCRIPTION
**Pull request by deontpearson**
## Description

Update readme as questions were asked

## Types of changes

_Put an `x` in the boxes that apply._

- [ ] Bugfix (`fix`): A non-breaking change which fixes an issue or bug
- [ ] New Feature (`feat`): A non-breaking change which adds a new feature or new functionality
- [ ] Tests (`test`): Adding missing tests or correcting existing tests
- [ ] Refactoring (`refactor`): A code change that neither fixes a bug nor adds a feature, e.g Deleting unused or redundant code
- [x] Documentation (`docs`): Documentation only changes
- [ ] Other (`style`, `build`, `ci`, `performance`): Formatting, linting or styling. Changes to the CI pipeline. Performance improvements. Changes that affect the build system.

https://uxi.atlassian.net/browse/BEP-737

### CODE REVIEW

File: charts/aruba-uxi/values.example.yaml [hunk 1]
```diff
@@ -224,6 +224,7 @@ aruba-uxi:
         enabled: false
         traceEnabled: false
         serviceName: example-service
+        env: {}
```
Comment (deontpearson) [inline charts/aruba-uxi/values.example.yaml:227]: A TODO for @MaraisKruger

File: charts/aruba-uxi/README.md [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
-### Value Files
+- `APPLICATION_NAME`: The name of the application, taken from `.Values.application`
+- `APPLICATION_VERSION`: The version of the application, taken from `.Values.image.tag`
+- `ENVIRONMENT`: The environment, taken from `.Values.global.environment`
+- `DD_ENABLED`: A boolean that can be used to control datadog (see `.Values.application.datadog.enabled`).
+- `DD_SERVICE`: The service name that datadog will use (see `.Values.application.datadog` for more).
+- `DD_ENV`: The environment name that datadog will use, taken from `Values.global.environment`.
+- `DD_TRACE_ENABLED`: A boolean that can be used to control datadog trace (see `.Values.application.datadog.traceEnabled`).
+- `SENTRY_ENVIRONMENT`: If `.Values.application.sentry.enabled`) this variable is set the application name.
+- `SENTRY_DSN`: If `.Values.application.sentry.enabled`) this variable is populate from the `.Values.application.sentry.dsn`.
```
Comment (MaraisKruger) [inline charts/aruba-uxi/README.md:24]: Don't we also want to add "DD_AGENT_HOST" and "DD_ENTITY_ID" as populated environment variables to this section?

Comment (deontpearson) [reply charts/aruba-uxi/README.md:24]: ah yes good catch

Comment (deontpearson) [reply charts/aruba-uxi/README.md:24]: ahh good catch, thanks

**Review by MaraisKruger — APPROVED** [review]

**Pull request merged**